### DISCUSSION TITLE
【招新】我的世界GrassCraft生草服务器招新啦

### DISCUSSION THREAD

**Discussion by yoimiya-shirokosdog** [share121/inter-knot#1400] [其他]
![2024-08-09_07 21 34](https://github.com/user-attachments/assets/d8323f4f-62fc-45aa-9c52-e5fe54a70523)
这是个生电建筑服，出生点周围资源丰富，环境优美，可以与你的好友一起生草
目前服务器处于前期，你可以自由探索
服务器地址：gc.9666.fun:18042
版本号：1.19.4
服务器无需正版验证
交流群号144274541

**Comment by EhuanyingK** [comment]
有多少人玩

**Comment by yoimiya-shirokosdog** [comment]
目前算上腐竹只有4人

**Comment by yoimiya-shirokosdog** [comment]
服务器注意事项：
1.目前服务器为云托管(绝不跑路)，偶尔会有维护现象，详情见交流群
2.服务器没有正版验证，只需首次进入时注册即可。后续进服登录，离开服务器4小时内无需再次登录
3.服务器有苦力怕防爆，无TNT防爆(可以捣鼓三向和世吞)，但请不要破坏他人房屋、建筑
4.大型生电机器开启需要报备
5.版本号为1.19.4，没有玩家间PVP
6.如果您无法连接至服务器，或没有信号(类似于在空洞、海底神殿、停电的办公室或学校)，可联系人工智能Fairy上门服务

**Comment by lllliufeng** [comment]
不会生电建筑怎么办

**Reply by yoimiya-shirokosdog** [reply]
养老即可

**Comment by Inter-Knot-User** [comment]
有圈地插件吗

**Comment by yoimiya-shirokosdog** [comment]
不会生电建筑，养老就好了

**Comment by yoimiya-shirokosdog** [comment]
这个插件没有

**Comment by EhuanyingK** [comment]
申请进群怎么没同意啊

**Comment by yoimiya-shirokosdog** [comment]
服务器现在接近打末影龙了，想要来的快来

**Comment by SteveNotSet** [comment]
这个委托我接了！

**Comment by RS64671012** [comment]
forge or fabric? @yoimiya-shirokosdog

**Comment by nostalgiatan** [comment]
抱歉，根据新定规则判定，您的帖子被判定为广告贴，按照新定规则，本应该删帖，但是因为该帖子发布时间期间并没有定下规则，所以不会按照新规则来判决，在此进行提醒和警告（无需回复，心知肚明即可）

**Comment by yoimiya-shirokosdog** [comment]
是插件端的，但是想要结合Fabric太难了

**Comment by yoimiya-shirokosdog** [comment]
放心这不是广子，这是个公益服，我拉几个新人而已
